packages/notebook/style/notebookfooter.css

```diff
@@ -2,20 +2,35 @@
  * Copyright (c) Jupyter Development Team.
  * Distributed under the terms of the Modified BSD License.
  */
-
-.jp-Notebook-footer {
-  height: 27px;
-  margin-left: calc(
-    var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
-      var(--jp-cell-padding) + var(--jp-notebook-padding)
+:root {
+  --jp-private-notebook-footer-width-spacer: calc(
+    100% -
+      (
+        var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) + 2 *
+          var(--jp-cell-padding) + 2 * var(--jp-notebook-padding)
+      )
   );
-  width: calc(
+  --dynamic-extra-width: clamp(58px, 7.87vw, 59.9px);
```

Comment:
This would need `--jp-private` prefix too. What do these values mean? I think we would need to find a definition which is derived from other variables, not based on just guessing the values.